client/src/app/mainwindow.cpp

```diff
@@ -428,6 +428,8 @@ void MainWindow::createTabs()
     mUi.tabWidget->addTab(page, i18n("&Opportunities"));
 
     connect(page, SIGNAL(modelCreated(ItemsTreeModel*)), this, SLOT(slotOppModelCreated(ItemsTreeModel*)));
+    connect(static_cast<AccountsPage*>(mAccountPage), &AccountsPage::requestNewOpportunity,
```

Comment:
Please change the type of mAccountPage to be AccountsPage so that no cast is necessary.
Casts are fragile when refactorings happen.

Reply:
done